Makefile

```diff
@@ -714,12 +714,13 @@ docker-binaries: clean
 enter:
 	make -C build.assets enter
 
-# grpc generates GRPC stubs from service definitions
+# grpc generates GRPC stubs from service definitions.
+# This target runs in the buildbox container.
 .PHONY: grpc
 grpc:
 	$(MAKE) -C build.assets grpc
 
-# buildbox-grpc generates GRPC stubs inside buildbox
+# buildbox-grpc generates GRPC stubs
```

Comment:
Note: I thought about renaming this to something less confusing (it doesn't run in the buildbox), but we have other repos that invoke `make buildbox-grpc` and I didn't want to break them.

Reply:
Perhaps worth adding a similar note in the comment itself so that a future reader isn't befuddled.